cli/cli_utils.go

```diff
@@ -188,15 +182,17 @@ func runScriptOrExit(flags *CliCmdFlags) (err error) {
 		os.Exit(0)
 	}
 
+	if err := buildContainers(*flags.services, flags.generationPath); err != nil {
```

Comment:
The main reason is that the `validator-import` service must be built before each execution with the `cli` command, to avoid using an old `validator-import` image for a different validator client. Not all validator clients need a custom `validator-import` image, but running `docker compose build` ensures that we are using the correct image if necessary. If the `validator-import` image for the current validator client is not customized, no image is built and Sedge process continues as normal.